test/index.js

```diff
@@ -105,6 +105,27 @@ describe('wt-tracker.', () => {
       done();
     });
   });
+  it('should handle after first load events', (done) => {
+    const events = [{hello: 'world', url: HREF}];
+    let touchedCount = 0;
+    wt('resetFirstLoad');
+    wt('afterFirstLoad', () => {
+      touchedCount ++;
+    });
+    assert.equal(touchedCount, 0);
+    runEvents(events, () => {
+      assert.equal(touchedCount, 1);
+      wt('afterFirstLoad', () => {
+        touchedCount ++;
+      });
+      assert.equal(touchedCount, 2);
+      done();
+    });
+    wt('resetFirstLoad');
+    wt('afterFirstLoad', () => {
+      touchedCount ++;
+    });
```

Comment:
Since there are no assertions after lines 124-27, what's the purpose of these lines? Just to clean up for the next test?

Reply:
the `done` call happens earlier in an async block, but it should fire *after* these two blocks right

Reply:
Ah, I see now - the callback passed to `runEvents` won't run until the call stack is empty. Thanks for clarifying!